aviatrix/resource_aviatrix_edge_spoke_transit_attachment.go

```diff
@@ -111,6 +111,18 @@ func resourceAviatrixEdgeSpokeTransitAttachment() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"dst_wan_interfaces": {
```

Comment:
Is "dst" here destination?  If so why does it have the concept of directionality?  Looking at the broader scope where we have `default_edge_wan_interfaces`, `edge_want_interfaces`, and now `dst_wan_interfaces` it seems quite confusing as a consumer of this.

Reply:
This is what Purnima mentioned - src interface is interface endpoint at the local gateway where the peering originates, and dst interface is the interface end point at the remote peer where this peering terminates.

Reply:
Removing this parameter from here in this PR.